src/logic/safe/store/actions/transactionsNew/loadAllTransactions.ts

```diff
@@ -72,6 +72,7 @@ const fetchAllTransactions = async (
 }
 
 const etagsByPage = {}
+let totalTransactionsAmount = null
```

Comment:
this number is unique for each safe but this action used for all safes, do we need really need to store it there?
could we maybe do something like
```
transactions: {
  safeAddress: {
    totalCount: 1000,
    data: []
  }
}
```

Reply:
It was done like that before and you asked me to change it

Reply:
@mikheevm do you have access to previous implementation? if so, could you please verify if former one was correct? just to avoid re-writing it in case we go back to it.

Reply:
https://github.com/gnosis/safe-react/blob/2888fd4494/src/logic/safe/store/reducer/allTransactions.ts

Here is the old implementation, it wasn't done like that

Reply:
In the old implementation totalTransactionsAmount was defined at top level instead of being specific to the safe

Reply:
check it now